consensus/obcpbft/pbft-core.go

```diff
@@ -192,6 +192,13 @@ func newPbftCore(id uint64, config *viper.Viper, consumer innerStack) *pbftCore
 	instance.idleChan = make(chan struct{})
 	instance.injectChan = make(chan func())
 
+	// TODO Ultimately, the timer factory will be passed in, and the existence of the manager
+	// will be hidden from pbftCore, but in the interest of a small PR, leaving it here for now
+	instance.manager = newEventManagerImpl(instance)
+	etf := newEventTimerFactoryImpl(instance.manager)
```

Comment:
There is already the `eventTimerFactory` interface (which sure, it's a very java name, and I'm open to changing it, suggestions welcome, though we have other `*Factory` interfaces in the code), and if we wanted to, we could have `eventManagerImpl` implement it, though I don't see why.

The idea would be that whatever is creating the timer, is who manages its interactions with the manager.  I see no strong reason as to why the state machines should have a reference to the event manager, in fact, I think unidirectional relationships are generally superior.  Ultimately, the `eventReceiver` is simply given a reference to an `eventTimerFactory`, with the simple contract that timers created with this factory will be processed through `processEvent` when they fire.  If at some point later we need some more sophisticated interaction between the `eventReceiver` `eventManager` and `eventTimer`, then we could add it, but I would keep it simple for now.